src/module3/config/constants.ts

```diff
@@ -1 +1,2 @@
 export const POSTGRES_URL='postgres://postgres:postgres@127.0.0.1:5432/postgres';
+export const JWT_SECRET='p4sta.w1th-b0logn3s3-s@uce';
```

Comment:
Interesting secret :)
Suggestion: Check for what qualifies as a good secret key.